components/AvalancheCenterLogo.tsx

```diff
@@ -94,48 +96,48 @@ export const AvalancheCenterLogo: React.FunctionComponent<AvalancheCenterLogoPro
     case 'NWAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/NWAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/NWAC.png')).uri);
     case 'MSAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/MSAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/MSAC.png')).uri);
     case 'MWAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/MWAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/MWAC.png')).uri);
     case 'PAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/PAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/PAC.png')).uri);
     case 'SNFAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/SNFAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/SNFAC.png')).uri);
     case 'SAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/SAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/SAC.png')).uri);
     case 'WCMAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/WCMAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/WCMAC.png')).uri);
     case 'CAIC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/CAIC.jpg')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/CAIC.jpg')).uri);
     case 'COAA':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/COAA.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/COAA.png')).uri);
     case 'CBAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/CBAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/CBAC.png')).uri);
     case 'ESAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/ESAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/ESAC.png')).uri);
     case 'WAC':
-      return Image.prefetch(Image.resolveAssetSource(require('../assets/logos/WAC.png')).uri);
+      return ImageCache.prefetch(queryClient, Image.resolveAssetSource(require('../assets/logos/WAC.png')).uri);
   }
   const invalid: never = avalancheCenter;
-  throw new Error(`Unknown avalanche center: ${invalid}`);
+  throw new Error(`Unknown avalanche center: ${JSON.stringify(invalid)}`);
```

Comment:
isn't `invalid` just a string?

Reply:
I don't know! I saw `[object Object]` :)

Reply:
that...should be impossible? well let me know what you see with this change!